### DISCUSSION TITLE
Access Seq from outside Amazon EC2 instance

### DISCUSSION THREAD

**Discussion by jangils** [datalust/seq-tickets#1733] [Help] [answered]
Hi, I'm struggling to access the Seq server from outside an Amazon EC instance. The interface runs fine on http://localhost:5341 from inside the EC instance. But I cannot access it from `http://ip-of-instance:5341`.

I've checked already the following steps:

- In the amazon console the security group is adjusted so it allows inbound traffic at port 5341 to 0.0.0.0
- In the windows firewall, the same port has been opened for inbound traffic
- I checked the subnet in AWS console, so it has a routetable from 0.0.0.0 to the default gateway to have internet access
- I changed the "listenUris" in seq.json to any combination of `http://0.0.0.0:5431`, `http://ip-of-instance:5341`, ... but none of them are working. If I use the combination of `http:localhost:5341` and `http://0.0.0.0:5341` or `http://ip-of-instance:5341`, the localhost is also not working anymore

I've read all the documentation and searched through the discussions here and on stackoverflow, but I cannot find a solution.
https://docs.datalust.co/docs/urls

The instance is running Windows Server 2022, and hosts a SaaS application (port 443) which runs fine and is accessible. Other port fowards are also working. But it seems I need to get Seq running on the address 0.0.0.0 instead of 127.0.0.1.

I also tried using a custom FQDN just for Seq with an A-record in the DNS settings, and using this FQDN as "listenUris". But this also didn't work. I checked  the FQDN, that it has the correct IP-address.

I don't know how to fix this, so any help would be greatly appreciated.

**Comment by nblumhardt** [comment]
Hi! Thanks for reaching out.

When you say:

> But I cannot access it from "http://:5341

do you mean "from `http://some-host:5341`"? Assuming so, but please let me know if otherwise.

Also please let me know if you're using HTTPS, since the issues there will differ.

In Seq's `listenUris` setting, just `http://localhost:5341` should be sufficient; `localhost` in this setting on Windows is treated as all-addresses and not loopback.

Hope this helps,
Nick

**Reply by jangils** [reply]
Sorry, my question was edited apperently. When I say `http://:5341` it should be `http://ip-adres-of-server:5341`

**Comment by jangils** [comment, marked as answer]
OK, stupid me... I detected an error in the Windows firewall rule. The issue is resolved.

**Reply by nblumhardt** [reply]
Great! Thanks for following up :+1:
